test/integration/order_integration_test.rb

```diff
@@ -10,9 +10,9 @@ def setup
                         first_name: 'John',
                         last_name: 'Doe',
                         email: 'example@example.com')
-    @item = Item.create(title: 'Pour Over', description: 'blah', price: 1000)
-    @order = item.orders.create(total_cost: 1000, user_id: user.id)
-    @order.item_orders.first.update(quantity: 1, line_item_cost: 1000)
+    @project = Project.create(title: 'Pour Over', description: 'blah', price: 1000)
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.